{{cookiecutter.project_slug}}/mkdocs.yml

```
@@ -0,0 +1,59 @@
+site_name: "{{cookiecutter.project_name}}"
+site_description: "Documentation website for {{cookiecutter.project_name}}"
+site_author: "{{cookiecutter.author_given_names}} {{cookiecutter.author_family_names}}"
+copyright: "Copyright © {% now 'utc', '%Y' %} {{cookiecutter.author_given_names}} {{cookiecutter.author_family_names}}"
+repo_url: "https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}/"
+repo_name: "{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}"
+edit_uri: edit/main/docs/
+
+validation:
+  omitted_files: warn
+  absolute_links: warn
+  unrecognized_links: warn
+
+theme:
+  name: "material"
+  features:
+    - content.action.edit
+  palette:
+    - media: "(prefers-color-scheme)"
+      toggle:
+        icon: material/brightness-auto
+        name: Switch to light mode
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      toggle:
+        icon: material/brightness-4
+        name: Switch to system preference
+  icon:
+    repo: fontawesome/brands/github
+
+nav:
+  - Overview: index.md
+  - API reference: api.md
+  - License: LICENSE.md
+
+markdown_extensions:
+  - pymdownx.tasklist
+  - pymdownx.snippets
+
+plugins:
+  - search
+  - mkdocstrings:
+      default_handler: python
+      handlers:
+        python:
+          docstring_style: google
+          import:
+            - "https://docs.python.org/3/objects.inv"
```

Quotes aren't necessary (I _think_)
```suggestion
            - https://docs.python.org/3/objects.inv
```

---

I think you're right in sense that it won't get misinterpreted by whatever YAML parser MkDocs is using, but there lots of [edge cases around processing of strings including colons `:`](https://stackoverflow.com/a/70909331) and quoting strings containing colons is [often recommended](https://stackoverflow.com/a/22235064). Noticed in writing this I didn't do this for `extra.social.link` definition below so will update that for consistency.

---

This is what I was wondering. But also anything in `-` becomes a string. Have learnt this from various battles with Ansible, and accidental double quoting (i.e. causing more issues than it solves).

---

You can have a colon separated key after a `-` as in the `- icon: fontawesome/brands/github` below so `- https://docs.python.org/3/objects.inv` could potentially be parsed equivalently to `- https: //docs.python.org/3/objects.inv` depending on the handling of spaces around colons.